### ISSUE TITLE
Disable warning squiggly lines

### ISSUE THREAD

**Issue by m1dnight** [elixir-lsp/elixir-ls#666]
### Precheck

* Do a quick search and make sure the issue has not yet been reported
* Do not use the issues tracker for guidance, questions or support (try Elixir Forum, Stack Overflow, Slack, etc. instead)
* If using a client other than VS Code, please try VSCode's "ElixirLS: Elixir support and debugger" extension. Does it reproduce your failure?

### Environment

* Elixir & Erlang versions (elixir --version): 
* Elixir Language Server version: 
* Operating system: 
* Editor or IDE name (e.g. Emacs/VSCode): 
* Editor Plugin/LSP Client name and version:

### Current behavior

The plugin underlines warnings that you usually get from the compiler, such as the following.

```
variable "stream" is unused (if the variable is not meant to be used, prefix it with an underscore)
```

I have a codebase that has a lot of these warnings, and it's supper annoying to try and program in a codebase that's full of these lines. 

### Expected behavior

Is it possible to add an option to disable these squiggly lines? Other language extensions seem to have that option: https://social.msdn.microsoft.com/Forums/sqlserver/en-US/b4278905-b34a-4396-945d-d7d661a29b21/how-to-hide-warning-scribblies-in-vs-code

Thanks for the plugin and all the hard work!

**Comment by lukaszsamson** [comment]
@m1dnight those warnings come from elixir compiler itself. For now disabling warnings is not supported both in elixir and in elixirLS. We could always accept a PR adding it.

**Comment by m1dnight** [comment]
But, the information is sent by the language server, correct? I am swamped the next month, but  I sure want to give this a shot if I can.

**Comment by lukaszsamson** [comment]
Yes, language server transforms warnings from the compiler and dialyzer and sends them via publishDiagnostic protocol message.
